lib/players/human.rb

```diff
@@ -0,0 +1,20 @@
+
+
+class Players
+
+    class Human < Player
+
+        def move(board, win = nil)
+            puts "Pick a spot"
+            input = gets
+            if input.match(/^(\d)+$/)
+            input
+        else
+            puts "invalid"
+            input = gets
+            input
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```